_data/projects/GitGud.yaml

```diff
@@ -0,0 +1,12 @@
+name: GitGud
+desc: A CLI git tool built using Ink, Node and React
+site: https://github.com/GitGud-org/GitGud
+tags:
+  - git
+  - cli
+  - ink
+  - reactjs
+  - node.js
+upforgrabs:
+  name: good first issue
+  link: https://github.com/GitGud-org/GitGud/projects/1?card_filter_query=label%3A%22good+first+issue%22
```

Comment:
@H3RSKO Please use this link https://github.com/GitGud-org/GitGud/labels/good%20first%20issue